vendor/assets/javascripts/spree/frontend/all.js

```diff
@@ -8,3 +8,4 @@
 //= require jquery_ujs
 //= require spree/frontend
 //= require_tree .
+
```

Comment:
I don't think you need this extra space at the bottom of the file, can you remove it please.